lib/routes/account.js

```diff
@@ -873,7 +874,7 @@ module.exports = function (
             id: isA.string().length(32).regex(HEX_STRING).required(),
             isCurrentDevice: isA.boolean().required(),
             lastAccessTime: isA.number().min(0).required().allow(null),
-            name: isA.string().max(255).required(),
+            name: isA.string().max(255).regex(DISPLAY_SAFE_UNICODE).required(),
```

Comment:
This will fail the request to fetch the list of devices if just one of them contains these unicode characters. That device list is used:

* To populate the devices view (obviously).
* But also by the browser [to self-resolve any potential device/session conflict](https://dxr.mozilla.org/mozilla-central/source/services/fxaccounts/FxAccounts.jsm#1567) (although don't know whether that state occurs in practice).

Not trying to be unhelpful, just wanted to check whether we want both of those things to fail in this case?